### DISCUSSION TITLE
give us electrician's goggles overlay in subeditor

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15088] [Ideas and suggestions]
Imagine if when we entered wiring mode or test mode, they were on by default, and with a checkbox to turn them on and off
it would be highly useful

edit: we have debugwiring command for test mode

**Comment by NotWendy** [comment]
To add to this, Electrician's Goggles are WAY too expensive to have a practical use in the campaign. You could remove fulgurium from the recipe and give these to the engineer by default. Give beginner engineers a chance to start learning wiring early, the same way medics get a Health Scanner HUD by default.
You guys have created a wonderful feature and then made it so impractical to access, I haven't seen anybody use it since it got added.

**Reply by girlyguppy** [reply]
this. it's not like it offers any advantage whatsoever, it simply makes wiring more intuitive. medical scanner is objectively WAY more powerful, and that is a beginner item

**Reply by pavel4win** [reply]
I vote with both hands - Electrician's Goggles must be in engineer's inventory at start and must have very simply recipe and must give a few materials during deconstruction.

**Comment by pavel4win** [comment]
In the editor, there is no current flowing through wires, so the electrician's glasses are useless. When running the submarine test, the electrician's glasses would be very useful.
IMHO it is necessary to add the electrician's glasses to the bots' inventory when running the submarine test from the editor.

**Reply by girlyguppy** [reply]
yes, you are correct about the editor. 
I propose a simulation mode where it loads only the components, and "tests", but keeps the player in wire editing mode with freecam and no lighting

**Comment by jigardi** [comment]
Have my upvote.
This would also make the wiring skills learned in the game more easily transferable to the editor.

Edit: Forgot a word.
